ui/src/app/app.component.ts

```diff
@@ -7,6 +7,6 @@ import { HttpClient } from '@angular/common/http';
   styleUrls: ['./app.component.css'],
 })
 export class AppComponent {
-  title = 'psd2-certificate-ui';
+  title = 'psd2-accelerator';
```

Comment:
Do we need both titles? (see index.html)

Reply:
According to https://angular.io/guide/set-document-title this should be the same value as in `<title>`, otherwise the title _should_ change back to _psd2-accelerator_ as soon as the app has loaded. Can we try this quickly tomorrow before we merge?